blocks/extensions/stopping.py

```diff
@@ -0,0 +1,35 @@
+from . import FinishAfter
+
+
+class FinishIfNoImprovementAfter(FinishAfter):
+    """Stop after improvements have ceased for a given period.
+
+    Parameters
+    ----------
+    iterations : int
```

Comment:
Maybe not necessary for this PR, but it'd be nice if we can support epochs as well, considering that the number of iterations per epoch can be variable.